### PR TITLE
Adds the blueshield (And just the blueshield) to the veteran crew list.

### PR DESCRIPTION
**Pull request by FlamingLily**
## About The Pull Request
gaming

## Why It's Good For The Game
After #6242, I've not seen a single person say adding the blueshield to the veteran crew list was bad, and most people agreed that blueshield is one of the roles that should very much be on there. Like Vanguards, it's not a security role, and veteran crew are trusted to not abuse their roles to validhunt. Obviously, I'm not policy team, but, I'm very confident this is the right play here. Especially considering the NT Rep PR.

## Changelog
idfk what to mark this as in my changelog that doesn't even work so I'm just not gonna and save my GBP 😎

### CODE REVIEW

**Review by Gandalf2k15 — APPROVED** [review]

**Comment by KathrinBailey** [comment]
@Gandalf2k15 you waiting on headmin approval or something or this good to go?

**Comment by Gandalf2k15** [comment]
Requires headmin approval, I give my +1.

**Comment by SpaceVampire** [comment]
Requires headmin approval, I give my +1.

**Comment by tf-4** [comment]
headmin, +1

**Comment by Gandalf2k15** [comment]
As this PR has received a majority vote from the head admins, it will now be merged.

**Pull request merged**
